mirdata/datasets/ballroom.py

```diff
@@ -130,6 +132,8 @@ def __init__(
         self.beats_path = self.get_path("beats")
         self.tempo_path = self.get_path("tempo")
 
+        self.genre = get_genre(self.audio_path)
```

Comment:
While making the changes I spotted a mistake. Here I changed line 135 to self.load_genre()

Reply:
Yes! Although are you sure is `self.load_genre()`? I think that the `load_genre()` function is outside the class right? Shouldn't it be without the `self.`?

Reply:
Oops, you are right! I assumed it was part of the class but I did not look carefully. Sorry about that!